tools/privacyidea-diag

```diff
@@ -138,18 +138,18 @@ pi_config() {
 	call_pi_manage resolver list -v >> "$pi_config"
 	log_pi_config "Realms"
 	log_pi_config "------"
-	call_pi_manage realm list >> "$pi_config"
+	call_pi_manage config export -t realm -f json >> "$pi_config"
```

Comment:
The original commands were just moved like
`call_pi_manage config realm list >> "$pi_config"`
The same works for the resolver list, it still censors the secret.

Reply:
thanks for the feedback. Done.